co30_Domination.Altis/x_macros.sqf

```diff
@@ -2,13 +2,22 @@
 //#define __DEBUG__
 
 #ifdef __DEBUG__
-#define __TRACE(tmsg) diag_log format ["T%1,DT%2,F%3,FPS%4,%5,%6,'%7'", time, diag_tickTime, diag_frameno, diag_fps, call {_xarsx = __FILE__ splitString "\";_xarsx select (count _xarsx - 1)}, __LINE__, tmsg];
-#define __TRACE_1(tmsg,parame1) diag_log format ["T%1,DT%2,F%3,FPS%4,%5,%6,'%7',%8: %9", time, diag_tickTime, diag_frameno, diag_fps, call {_xarsx = __FILE__ splitString "\";_xarsx select (count _xarsx - 1)}, __LINE__, tmsg, parame1, call compile format ["%1", parame1]];
-#define __TRACE_2(tmsg,parame1,parame2) diag_log format ["T%1,DT%2,F%3,FPS%4,%5,%6,'%7',%8: %9,%10: %11", time, diag_tickTime, diag_frameno, diag_fps, call {_xarsx = __FILE__ splitString "\";_xarsx select (count _xarsx - 1)}, __LINE__, tmsg, parame1, call compile format ["%1", parame1], parame2, call compile format ["%1", parame2]];
-#define __TRACE_3(tmsg,parame1,parame2,parame3) diag_log format ["T%1,DT%2,F%3,FPS%4,%5,%6,'%7',%8: %9,%10: %11,%12: %13", time, diag_tickTime, diag_frameno, diag_fps, call {_xarsx = __FILE__ splitString "\";_xarsx select (count _xarsx - 1)}, __LINE__, tmsg, parame1, call compile format ["%1", parame1], parame2, call compile format ["%1", parame2], parame3, call compile format ["%1", parame3]];
+

```

Comment:
what is this part?

Reply:
No functional impact.
while I tried to understand the code and the dependencies, I've reduced the TRACE output for easier reading.
Original code is not deleted, just commented.